docere/render.py

```diff
@@ -58,14 +60,32 @@ def tmp_cd(path):
         os.chdir(curdir)
 
 
+def get_metadata_generators():
+    default = [build_index]
+
+    try:
+        module = SourceFileLoader(
+            'local_config',
+            '.docere_config.py'
+        ).load_module()
```

Comment:
I think this will execute any code in the module; is that your intent?

Reply:
It is. I picked this trick up from [pelican](https://github.com/getpelican/pelican/blob/master/pelican/settings.py#L226). They avoid having to build a config language by using a local python file. This is especially handy since this PR allows the user to inject new plugins for generating metadata pages. 

This does open up some risk if your CI is building PRs by default. However, we've mitigated these issues in the existing docere KRs (i.e. mpr and mozilla-reports).